start-a-local-cluster.md

```diff
@@ -32,34 +34,38 @@ Data is stored on-disk and client/server communication is secure. Setup involves
     ```
 
 3. [Run some queries](basic-sql-commands.html).
-4. Check out the Admin UI by pointing your browser to `http://<your local host>:26257`. You can find your local host by running `hostname` in your shell.    
-
-## Standard Mode
 
-1. From the directory containing the `cockroach` binary, initialize the cluster:
+4. Check out the Admin UI by pointing your browser to `http://<your local host>:26257`. You can find your local host by running `hostname` in your shell.    
 
+5. To simulate a multi-node cluster, add each new node as follows:
+    
     ```bash
-    $ ./cockroach init --stores=ssd=dev/data
+    $ ./cockroach start --dev --port=26258 --pgport=15433 --join=<your local host>:26257
```

Comment:
This doesn't currently work – you're only creating two separate dev mode clusters. They're not talking to each other. I don't think we should support using --dev mode in this way.

I'm going to submit some fixes for this.

You should go back to have a Standard mode like we used to have.